test/libdnf/system/test_state.cpp

```diff
@@ -63,6 +63,16 @@ enabled_stream = "stream-1"
 
 )"""};
 
+static inline void erase_newlines(std::string & str) {
+    str.erase(std::remove(str.begin(), str.end(), '\n'), str.cend());
```

Comment:
Does this remove all newlines? You don't want to do that... The previous version was better, though I'd only trim the trailing newlines, as this is a workaround and ideally you don't want any stray newlines at either end, the extraneous newline should actually ideally be fixed in `toml11`... (I haven't reported a bug for it yet)

Reply:
OK I wanted to simplify things here. The problem is there is/is not a trailing newline between each section, so for the `modules_contents` case, the trailing remove is not enough.